terraform/browser/browser.tf.json.template.py

```diff
@@ -48,6 +48,64 @@
 provision_custom_search = False
 
 
+def q(s: str) -> str:
+    """
+    Return the string wrapped in single quotes
+    """
+    return f"'{s}'"
+
+
+def s(*args: str) -> str:
+    """
+    Return the arguments as a space-separated string
+    """
+    return ' '.join(args)
+
+
+def content_security_policy() -> str:
+    return ';'.join([
+        s('default-src', q('self')),
```

Comment:
Extract variable `self` so that

```suggestion
        s('default-src', self),
```

Reply:
Same with `none`